### PR TITLE
iNOG::15v updates across the board inc waiting list

### PR DESCRIPTION
**Pull request by donalod**
Y NO U AUTO MERGE

### CODE REVIEW

File: pub/host/index.html [hunk 1]
```diff
@@ -166,8 +166,8 @@ <h5>uses Paypal</h5>
 
       <div class="row">
         <div class="col-lg-6 col-md-8 col-sm-12 col-xs-12 col-lg-offset-3 col-md-offset-2">
-          <a href="https://goo.gl/forms/Qa9zS5UjqHNxNmo52" target="_blank"><span class="label label-success" style="font-size: 1.0em;text-decoration: underline;">iNOG::14v (online/virtual only)</span></a> - Late <b>April 2020</b> - Fri 4pm</br>
-          <a href="https://goo.gl/forms/Qa9zS5UjqHNxNmo52" target="_blank"><span class="label label-default" style="font-size: 1.0em;text-decoration: underline;">iNOG::15v</span></a> - Late <b>June/July 2020</b> - Fri 4pmpm</br>
+          <a href="https://goo.gl/forms/Qa9zS5UjqHNxNmo52" target="_blank"><span class="label label-success" style="font-size: 1.0em;text-decoration: underline;">iNOG::15v (online/virtual only)</span></a> - Late <b>May 2020</b> - Fri 4pmpm</br>
```
Comment (cmsirbu) [inline pub/host/index.html:169]: small typo here and next row - `4pmpm`

File: pub/live/index.html [hunk 2]
```diff
@@ -52,7 +52,7 @@
       <div class="navbar-collapse collapse" id="navbar-main">
         <ul class="nav navbar-nav navbar-right">
           <!-- <li><a class="btn btn-link" href="people.html">People</a></li> -->
-          <li><a class="btn btn-link" href="/live" data-toggle="tooltip" data-placement="bottom" title="Watch Live">Live <i class="fas fa-video"></i> 4pm (IST) Fri 17th April</i></a></li>
+          <li><a class="btn btn-link" href="/live" data-toggle="tooltip" data-placement="bottom" title="Watch Live">Live <i class="fas fa-video"></i> 4pm (IST) Fri 29th May</i></a></li>
```
Comment (cmsirbu) [inline pub/live/index.html:55]: Make this change on the main page navbar as well please, it still has the old date.

File: pub/host/index.html (hunk 1 above)
Comment (donalod) [reply pub/host/index.html:169]: Fixed.

File: pub/live/index.html (hunk 2 above)
Comment (donalod) [reply pub/live/index.html:55]: Fixed.

**Pull request merged**